pom.xml

```diff
@@ -341,7 +341,7 @@
         <plugin>
           <groupId>org.springframework.boot</groupId>
           <artifactId>spring-boot-maven-plugin</artifactId>
-          <version>2.0.4.RELEASE</version>
+          <version>2.1.6.RELEASE</version>
```

Comment:
Nice catch. But could you be so nice to use `${spring.boot.version}` so we get this aligned also in the future.

Reply:
> @vapadwal 👍 thanks. You solved the issue correct. We could already merge it this way. However, I added some comments for further improvements. As they should be trivial to implement, I would love to ask you to improve this PR.
> 
> Also we should add more tests (e.g. deserialization of Page, Pageable, Instant, Year, CsrfToken, etc.). I am fine if you can not include them on time into this PR but could you then be so kind to file an issue to add more tests for ObjectMapperFactory.

Issue added
https://github.com/devonfw/devon4j/issues/112